packages/login_client/lib/src/login_client.dart

```diff
@@ -51,7 +51,7 @@ class LoginClient extends http.BaseClient {
   /// to correctly restore saved credentials from the `credentialsStorage`.
   ///
   /// See also:
-  /// - [InMemoryCredentialsStorage]
```

Comment:
Because of [this lint](https://dart-lang.github.io/linter/lints/comment_references.html)

<img width="932" alt="Screenshot 2021-11-28 at 3 06 25 PM" src="https://user-images.githubusercontent.com/40357511/143771253-8ef08d60-dc82-456e-9504-f7bffc16e80c.png">

Reply:
fix'd:)